documentation/client-side-forms/tutorial-binder.asciidoc

```diff
@@ -6,11 +6,21 @@ layout: page
 
 = Binding Data to Client-Side Forms
 
-Vaadin 17+ provides means for binding UI components in TypeScript form views. There is a client-side `Binder` class that is responsible for keeping the state of the form, the default and current values, and validation of the data.
+Vaadin 17+ provides means for binding UI components in TypeScript form views.
```

Comment:
As a general guidance, avoid writing about specific Vaadin versions. Always assume you are writing about the latest/current version. The user has already chosen the documentation version they are reading, we don’t need to remind them again.
```suggestion
Vaadin provides means for binding UI components in TypeScript form views.
```